ega-charts/cega/values.yaml

```diff
@@ -10,7 +10,7 @@ persistence:
   enabled: true
 
 secrets:
-  cega_creds: ""
+  cega_creds: "cega"
```

Comment:
The correct syntax for this parameter is `username:password`  that `:` should be there.

Reply:
Seems to work like this, so nevermind :)